msteams-platform/index.yml

```
@@ -14,100 +15,120 @@ metadata:
-          itemType: concept
-          text: Understand Microsoft Teams use cases
-        - url:  /microsoftteams/platform/concepts/design/map-use-cases
+          text: IT Professionals
+        - url: https://support.microsoft.com/en-us/office/apps-and-services-cc1fba57-9900-4634-8306-2360a40c665b?ui=en-US&rs=en-US&ad=US#PickTab=Basics
           itemType: concept
+          text: End users
+
+   # Card
+    - title: Explore
+      links:
+        - url: /microsoftteams/platform/concepts/capabilities-overview
+          itemType: overview
+          text: Teams app capabilities
+        - url: /microsoftteams/platform/concepts/design/map-use-cases
+          itemType: overview
           text: Map use cases to Microsoft Teams capabilities
-        - url:  /microsoftteams/platform/whats-new 
-          itemType: whats-new
-          text: What's new
+        - url: /microsoftteams/platform/tutorials/get-started-nodejs-app-studio
+          itemType: how-to-guide
+          text: Build a Teams app using JavaScript/Node.js and App Studio
+        - url: /microsoftteams/platform/tutorials/get-started-dotnet-app-studio
+          itemType: how-to-guide
+          text: Build a Teams app using C#/.NET and App Studio
+        - url: /microsoftteams/platform/samples/teams-low-code-solutions
+          itemType: how-to-guide
+          text: Build low-code apps, bots, and flows with Microsoft Power Platform
+        - url:  /microsoftteams/platform/resources/schema/manifest-schema
+          itemType: reference
+          text: Manifest schema for Microsoft Teams
       
     # Card
     - title: Learn
       links:
         - url: /learn/modules/intro-microsoft-teams-apps
           itemType: learn
-          text: Build apps for Microsoft Teams — introduction
+          text: Build apps for Microsoft Teams
         - url: /learn/modules/embedded-web-experiences
           itemType: learn
-          text: Tabs — create embedded web experiences 
+          text: Create embedded web experiences with tabs
         - url:  /learn/modules/msteams-conversation-bots/
           itemType: learn
-          text: Bots — create conversational interactions
+          text: Create conversational interactions with bots
         - url: /learn/modules/msteams-messaging-extensions/
           itemType: learn
-          text: Messaging extensions — create task-oriented interactions 
+          text: Create task-oriented interactions with messaging extensions
         - url: /learn/modules/msteams-task-modules/
           itemType: learn
-          text: Task modules — collect input and gather information
+          text: Collect input and gather information with task modules
         - url: /learn/modules/msteams-webhooks-connectors/
           itemType: learn
-          text: Webhooks and Office 365 connectors — connect web services to channels and teams
+          text: Connect web services to channels and teams with webhooks and Office 365 connectors
```

```suggestion
          text: Connect web services to channels and teams with webhooks and Office 365 Connectors
```
"Connectors" is capitalized here since it is a product name.